src/ManageCourses.Domain/DatabaseAccess/ManageCoursesDbContext.cs

```diff
@@ -300,6 +300,24 @@ and lower(ou.email)=lower(@email)", new NpgsqlParameter("instCode", instCode), n
 
             return ucasCourses;
         }
+
+        public List<UcasCourse> GetUcasCourseRecordsByUcasCode(string instCode, string ucasCode)
```

Comment:
reiterating, this code duplication is an unnecessary pattern

reject

Reply:
Data service and DB context rolled back to master